test/dummy/db/migrate/20190114100127_create_session_histories.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class CreateSessionHistories < MIGRATION_CLASS
```

Comment:
We will also need to define some mongo schemas for this now.

Reply:
Yes, I added mongo model.